### PR TITLE
Add org.cvfosammmm.Lemma

### PR DESCRIPTION
**Pull request by cvfosammmm**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by cvfosammmm** [comment]
bot, build org.cvfosammmm.Lemma

File: org.cvfosammmm.Lemma.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        --prefix=${FLATPAK_DEST} "pyxdg" --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/e5/8d/cf41b66a8110670e3ad03dab9b759704eeed07fa96e90fdc0357b2ba70e2/pyxdg-0.28-py2.py3-none-any.whl
+        sha256: bdaf595999a0178ecea4052b7f4195569c1ff4d344567bccdc12dfdf02d545ab
+  - name: python3-uharfbuzz
+    buildsystem: simple
+    build-options:
+        build-args:
+          - --share=network
```
Comment (hfiguiere) [inline org.cvfosammmm.Lemma.yml:31]: this is not allowed. build must be offline.

File: org.cvfosammmm.Lemma.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: lemma
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
```
Comment (hfiguiere) [inline org.cvfosammmm.Lemma.yml:12]: why host?

File: org.cvfosammmm.Lemma.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  - --env=PATH=/app/bin:/usr/bin:/app/bin/x86_64-linux
```
Comment (hfiguiere) [inline org.cvfosammmm.Lemma.yml:13]:
```suggestion
```

unnecessary

File: org.cvfosammmm.Lemma.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  - --env=LC_ALL=C
+  - --talk-name=ca.desrt.dconf
```
Comment (hfiguiere) [inline org.cvfosammmm.Lemma.yml:15]: are you sure you need dconf or is this just cut & paste?

File: org.cvfosammmm.Lemma.yml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+command: lemma
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --env=PATH=/app/bin:/usr/bin:/app/bin/x86_64-linux
+  - --env=LC_ALL=C
```
Comment (hfiguiere) [inline org.cvfosammmm.Lemma.yml:14]: why forcing the locale to C?

File: org.cvfosammmm.Lemma.yml (hunk 4 above)
Comment (cvfosammmm) [reply org.cvfosammmm.Lemma.yml:15]: the latter.

File: org.cvfosammmm.Lemma.yml (hunk 2 above)
Comment (cvfosammmm) [reply org.cvfosammmm.Lemma.yml:12]: It's currently not needed. What would you recommend?

File: org.cvfosammmm.Lemma.yml (hunk 1 above)
Comment (cvfosammmm) [reply org.cvfosammmm.Lemma.yml:28]: I couldn't manage to build it without this arg. Any ideas?

File: org.cvfosammmm.Lemma.yml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+sdk: org.gnome.Sdk
+command: lemma
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
```
[the remark this answers is not in the document]
Comment (cvfosammmm) [reply org.cvfosammmm.Lemma.yml]: removed

File: org.cvfosammmm.Lemma.yml [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: lemma
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
```
[the remark this answers is not in the document]
Comment (cvfosammmm) [reply org.cvfosammmm.Lemma.yml]: removed

File: org.cvfosammmm.Lemma.yml (hunk 1 above)
Comment (hfiguiere) [reply org.cvfosammmm.Lemma.yml:26]: any dependencies should be installed
we have some tooling for that https://github.com/flatpak/flatpak-builder-tools

But there is no information you provided to help you further.

This apply to other modules too.

File: org.cvfosammmm.Lemma.yml (hunk 2 above)
Comment (cvfosammmm) [reply org.cvfosammmm.Lemma.yml]: removed

File: org.cvfosammmm.Lemma.yml (hunk 1 above)
Comment (cvfosammmm) [reply org.cvfosammmm.Lemma.yml:25]: The pip tool doesn't work.

This is the output for the fontconfig module:
`========================================================================
Downloading sources
========================================================================
Running: "pip3 download --exists-action=i --dest /tmp/pip-generator-python3-fontconfig-pyur0_0i8o -r /tmp/requirements.ahv2vw41"
DEPRECATION: Loading egg at /usr/local/lib/python3.11/dist-packages/Python_FreeType-0.62-py3.11.egg is deprecated. pip 24.3 will enforce this behaviour change. A possible replacement is to use pip for package installation.. Discussion can be found at https://github.com/pypa/pip/issues/12330
Collecting fontconfig-py (from -r /tmp/requirements.ahv2vw41 (line 1))
  Using cached fontconfig_py-0.1.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (1.4 kB)
Using cached fontconfig_py-0.1.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.3 MB)
Saved /tmp/pip-generator-python3-fontconfig-pyur0_0i8o/fontconfig_py-0.1.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
Successfully downloaded fontconfig-py
========================================================================
Downloading arch independent packages
========================================================================
Traceback (most recent call last):
  File "/home/hp/Setzer/flatpak/pip/./flatpak-pip-generator", line 291, in <module>
    url = get_tar_package_url_pypi(name, version)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hp/Setzer/flatpak/pip/./flatpak-pip-generator", line 91, in get_tar_package_url_pypi
    raise Exception(err)
Exception: Failed to get fontconfig_py-0.1.2 source from https://pypi.org/pypi/fontconfig_py/0.1.2/json
`

**Comment by cvfosammmm** [comment]
bot, build org.cvfosammmm.Lemma

**Comment by cvfosammmm** [comment]
bot, build org.cvfosammmm.Lemma

File: org.cvfosammmm.Lemma.yml (hunk 1 above)
Comment (cvfosammmm) [reply org.cvfosammmm.Lemma.yml]: No, the harfbuzz build fails, see https://buildbot.flathub.org/#/builders/26/builds/12298

The manifest was generated with `flatpak-pip-generator`.

**Comment by cvfosammmm** [comment]
bot, build org.cvfosammmm.Lemma

**Comment by cvfosammmm** [comment]
bot, build org.cvfosammmm.Lemma

**Comment by cvfosammmm** [comment]
bot, build org.cvfosammmm.Lemma

**Comment by cvfosammmm** [comment]
bot, build org.cvfosammmm.Lemma

**Comment by cvfosammmm** [comment]
bot, build org.cvfosammmm.Lemma

**Comment by cvfosammmm** [comment]
bot, build org.cvfosammmm.Lemma

File: org.cvfosammmm.Lemma.yml (hunk 1 above)
Comment (cvfosammmm) [reply org.cvfosammmm.Lemma.yml]: I remove the modules that couldn't be built.

**Comment by cvfosammmm** [comment]
Is there still something missing? @hfiguiere

File: org.cvfosammmm.Lemma.yml (hunk 1 above)
Comment (hfiguiere) [reply org.cvfosammmm.Lemma.yml]: what were they needed for then?

Comment (cvfosammmm) [reply org.cvfosammmm.Lemma.yml]: I have alternatives to the libraries now directly in the Lemma repo. That's why I could remove them.

**Comment by cvfosammmm** [comment]
@bbhtt are you still awaiting changes?

**Comment by bbhtt** [comment]
No seems fine

**Comment by cvfosammmm** [comment]
Thanks :)

**Pull request closed without merging**